src/components/Dashboard/Form/DocumentForm.scss

```diff
@@ -5,8 +5,20 @@
   padding: 20px 40px;
   height: 84%;
   @include containerStyle();
+
 }
 
 :global(.FormContent) {
   padding-left: 90px;
 }
+div .buttons{
+  //@include flex(wrap);
+  position: absolute;
+  bottom: 93px;
+  right: 130px;
+  left: 1080px;
```

Comment:
DON'T DO IT :D don't use position absolute when you don't need it, there is a looot better methods to position it in the same way :) flex?

Reply:
I changed it for 'flex' but I still have a problem with position of button"dalej" on the first form (is on the left)